spec/rubocop/cop/performance/regexp_match_spec.rb

```diff
@@ -85,13 +85,13 @@ def foo
           do_something
         end
       end
-    RUBY
+                     RUBY
       def foo
         if #{correction}
           do_something
         end
       end
-    RUBY2
+                     RUBY2
```

Comment:
I think that this cop is useful 👍, but It may be strange to indent more than a strings written in a heredoc.

Reply:
I agree! I'd love advice about what this block should look like. I'd sort of expected the autocorrector to correct the opening heredocs to move them to the left
```ruby
include_examples :offense, "#{name} in method definition",
  <<-RUBY
    def foo
    end
  RUBY
```
or to move the code inside the heredoc to the right with the closing heredoc
```ruby
include_examples :offense, "#{name} in method definition",
                 <<-RUBY
                   def foo
                   end
                 RUBY
```
. What do we think should happen in this case?

Reply:
I'm not sure, but currently is a bit weird. As a general rule, I don't think the delimiter should ever be indented further than the content. Would you agree?

Reply:
Hm, I worried this change and I opened a PR #5943 😃